src/Symfony/Component/Console/Input/InputInterface.php

```diff
@@ -74,11 +72,13 @@ public function validate();
     public function getArguments();
 
     /**
-     * Gets argument by name.
+     * Returns the argument value for a given argument name.
+     *
+     * @param string $name The argument name
      *
-     * @param string $name The name of the argument
+     * @return mixed The argument value
      *
-     * @return mixed
+     * @throws \InvalidArgumentException When argument given doesn't exist
```

Comment:
In fact the exception class is `Console\Exception\InvalidArgumentException`, same for other.
Do we add a use statement in this case? I used the parent exception so it stays correct though less precise.